lib/things.py

```diff
@@ -16,8 +16,7 @@ def create(self, thing, token):
             mf_resp.error.status = 1
             mf_resp.error.message = errors.handle_error(errors.things["create"], http_resp.status_code)
         else:
-            location = http_resp.headers.get("location")
-            mf_resp.value = location.split('/')[2]
+            mf_resp.value = http_resp.json()
```

Comment:
This is not correct. Whe only one thing is created, response is not JSON but just HTTP status and Thing ID in the location header.

Reply:
Please fix this in the API doc as well (send PR to docs).

You can use this API reference as well: https://api.mainflux.io/?urls.primaryName=things.yml#/things/post_things, generated out of Swagger YAML files.

BUt I noticed there one error as well, related to response header. This should be corrected as well (in the same PR).

Reply:
This can be closed as I have submitted a pull request on the docs repo [here](https://github.com/mainflux/docs/pull/120/)